update docs: installation, processing pipeline, ML components (#513)
POSYDON-code/POSYDON: docs/_source/components-overview/post_processing/pipeline_ini.rst

        EMAIL = 'matthias.kruckow@unige.ch'
        GROUP = 'GL_S_Astro_POSYDON'

The last one :samp:`GROUP` is a bit special. If it is set, ownership of all the files

> created by the pipeline will be assigned to :sampl:`GROUP`. This is especially helpful if more than one

Reviewer: created by the pipeline will be assigned to :sampl:`GROUP`. This is especially helpful if more than one
```suggestion
created by the pipeline will be assigned to :samp:`GROUP`. This is especially helpful if more than one
```

> The path of each grid will be joint as

Reviewer 2: ```suggestion
The path of each grid will be joined as
```

    COMPRESSIONS        a list of lists of compression types
    DROP_MISSING_FILES  boolean to ignore missing files; it allows to have different substructures and taking the existing ones into account when specifying union of all possible substructures
    CREATE_PLOTS        a list of plots to make; this will be done independently whether the step is active or not; to make no plots put there an empty list or comment out such a line

>     DO_CHECKS           a list of checks to perform; this will be done independently whether the step is active or not; to omitt checks specify an empty list or comment out the line

Reviewer 2: ```suggestion
    DO_CHECKS           a list of checks to perform; this will be done independently whether the step is active or not; to omit checks, specify an empty list or comment out the line
```

    ====  ============================  ===========
    Step  Keyword                       Description
    ====  ============================  ===========

>        1  STOP_BEFORE_CARBON_DEPLETION  indicating, whether high mass HMS stars should get their history cropped short before carbon depletion (1) or not (0)

Reviewer 2: ```suggestion
       1  STOP_BEFORE_CARBON_DEPLETION  indicating, whether high mass HMS stars should get their history cropped before carbon depletion (set to 1 if so) or not (set to 0 if not)
```

       2  GRID_SLICES                   for this step, we have 3 layers of lists: the outermost is still the grid type, the inner most is still the grid slice, the middle layer is the combined grid
       2  GRIDS_COMBINED                a list of lists of combined grids; the outermost list is again referring to grid type; this is used as name for the new combined grid instead of :samp:`GRID_SLICE`
       3  ORIGINAL_COMPRESSIONS         a list of lists of the ORIGINAL compression to calculate the extra values from (the first one is used for all compressions for that grid type)
       4  INTERPOLATION_METHODS         a list of the interpolator types which are trained

>        4  CONTROL_GRIDS                 a list of lists of control grids for the :samp:`GRID_SLICES`; it need to have the same number of entries as the :samp:`GRID_SLICES`, to specify no control grid use an empty string

Reviewer 2: ```suggestion
       4  CONTROL_GRIDS                 a list of lists of control grids for the :samp:`GRID_SLICES`; it needs to have the same number of entries as the :samp:`GRID_SLICES`; use an empty string to specify no control grid
```
